glastopf/glastopf.cfg.dist

```diff
@@ -87,3 +87,9 @@ handler = AMQP/TCP/UDP
 [misc]
 # set webserver banner
 banner = Apache/2.0.48
+
+[surface]
+#https://www.google.com/webmasters/
+google_meta =
+#http://www.bing.com/toolbox/webmaster
```

Comment:
@glaslos :  shall we remove it ? I thought this will help the users to spot he google bing meta tools easily.